static/app/views/replays/detail/accessibility/index.tsx

```diff
@@ -1,13 +1,272 @@
+        data-test-id="replay-details-accessibility-tab"
+      >
+        <SplitPanel
+          style={{
+            gridTemplateRows: splitSize !== undefined ? `1fr auto ${splitSize}px` : '1fr',
+          }}
+        >
+          {accessibilityData ? (
+            <OverflowHidden>
+              <AutoSizer onResize={onWrapperResize}>
+                {({height, width}) => (
+                  <MultiGrid
+                    ref={gridRef}
+                    cellRenderer={cellRenderer}
+                    columnCount={COLUMN_COUNT}
+                    columnWidth={getColumnWidth(width)}
+                    deferredMeasurementCache={cache}
+                    estimatedColumnSize={100}
+                    estimatedRowSize={BODY_HEIGHT}
+                    fixedRowCount={1}
+                    height={height}
+                    noContentRenderer={() => (
+                      <NoRowRenderer
+                        unfilteredItems={accessibilityData}
+                        clearSearchTerm={clearSearchTerm}
+                      >
+                        {t('No accessibility problems detected')}
+                      </NoRowRenderer>
+                    )}
+                    onScrollbarPresenceChange={onScrollbarPresenceChange}
+                    onScroll={() => {
+                      if (scrollToRow !== undefined) {
+                        setScrollToRow(undefined);
+                      }
+                    }}
+                    scrollToRow={scrollToRow}
+                    overscanColumnCount={COLUMN_COUNT}
+                    overscanRowCount={5}
+                    rowCount={items.length + 1}
+                    rowHeight={({index}) => (index === 0 ? HEADER_HEIGHT : BODY_HEIGHT)}
+                    width={width}
+                  />
+                )}
+              </AutoSizer>
+            </OverflowHidden>
+          ) : (
+            <Placeholder height="100%" />
+          )}
+          {/* <AccessibilityDetails
+            {...resizableDrawerProps}
+            item={detailDataIndex ? items[detailDataIndex] : null}
+            onClose={() => {
+              setDetailRow('');
+            }}
+            projectId="1"
+            startTimestampMs={startTimestampMs}
+          /> */}
+        </SplitPanel>
+      </AccessibilityTable>
+    </FluidHeight>
+  );
 }
 
-export default A11y;
+const SplitPanel = styled('div')`
+  width: 100%;
+  height: 100%;
+
+  position: relative;
+  display: grid;
+  overflow: auto;
+`;
+
+const OverflowHidden = styled('div')`
+  position: relative;
+  height: 100%;
+  overflow: hidden;
+  display: grid;
+`;
+
+const AccessibilityTable = styled(FluidHeight)`
+  border: 1px solid ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
+
+  .beforeHoverTime + .afterHoverTime:before {
+    border-top: 1px solid ${p => p.theme.purple200};
+    content: '';
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 999999999%;
```

Comment:
Does this behave differently than `100%`?

Reply:
yes! 100% would be the width of the column, this is set to be really/arbitrarily wide so it crosses under all columns, starting at the left edge of the table (left: 0, the relative parent is the container of the table)

It's fully copied from network and errors tabs, this is the 3rd example which means time to refactor in a followup. There are more things to refactor too